hummingbot/market/market_base.pyx

```diff
@@ -194,6 +194,9 @@ cdef class MarketBase(NetworkIterator):
         order_size_quantum = self.c_get_order_size_quantum(trading_pair, amount)
         return (amount // order_size_quantum) * order_size_quantum
 
+    def supported_order_types(self):
+        raise NotImplementedError
```

Comment:
Sorry, after some consideration, I think it'd be better if this returns the common MARKET and LIMIT order types so that paper trade and DEX markets will have these automatically.

Reply:
Done!